modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/action/executor/FunctionActionExecutorImpl.java

```diff
@@ -171,42 +172,47 @@ else if (jsonObject.has("serializable")) {
 					"There are no workflow task assignees for Kaleo action " +
 						kaleoAction);
 			}
-
-			return;
 		}
 
+		payloadJSONObject.put(
+			"entryDTO",
+			_getEntryDTOJSONObject(
+				(String)inputObjects.get(
+					WorkflowConstants.CONTEXT_ENTRY_CLASS_NAME),
+				GetterUtil.getLong(
+					inputObjects.get(WorkflowConstants.CONTEXT_ENTRY_CLASS_PK)),
+				executionContext));
+
+		launch(payloadJSONObject, workflowTaskAssignees);
+	}
+
+	protected void launch(
+			JSONObject payloadJSONObject,
+			List<WorkflowTaskAssignee> workflowTaskAssignees)
+		throws Exception {
+
 		WorkflowTaskAssignee workflowTaskAssignee = workflowTaskAssignees.get(
 			0);
 
+		long userId = workflowTaskAssignee.getAssigneeClassPK();
```

Comment:
I think that if the `workflowTaskAssignees` comes empty, the `workflowTaskAssignee` variable will be null and then we will have an NPE at this point, am I right?